src/components/SearchResults/SearchResults.js

```diff
@@ -338,68 +361,107 @@ const SearchResults = () => {
-                    <FiltersContainer>
-                      <FilterHeader>{specifySearchText}</FilterHeader>
-                      <StyledSearchFilters />
-                    </FiltersContainer>
+                    {!searchFinished ? (
+                      <>
+                        <StyledSearchResults>
+                          {[...Array(10)].map((_, index) => (
+                            <StyledLoadingSkeletonContainer key={index}>
+                              <Skeleton borderRadius={SKELETON_BORDER_RADIUS} width={200} />
+                              <Skeleton borderRadius={SKELETON_BORDER_RADIUS} />
+                              <Skeleton count={2} borderRadius={SKELETON_BORDER_RADIUS} inline width={60} />
+                            </StyledLoadingSkeletonContainer>
+                          ))}
+                        </StyledSearchResults>
+                        <FiltersContainer>
+                          <FilterHeader>{specifySearchText}</FilterHeader>
+                          <Skeleton count={2} borderRadius={SKELETON_BORDER_RADIUS} width={200} />
+                        </FiltersContainer>
+                      </>
+                    ) : (
+                      <>
+                        <EmptyResultsContainer
+                          css={css`
+                            grid-area: results;
+                            margin-top: 80px;
+                          `}
+                        >
+                          <EmptyResults />
+                        </EmptyResultsContainer>
+                        <FiltersContainer>
+                          <FilterHeader>{specifySearchText}</FilterHeader>
+                          <StyledSearchFilters />
+                        </FiltersContainer>
+                      </>
+                    )}
                   </>
                 )}
+                {showMobileFilters && isTabletOrMobile && <MobileFilters />}
+              </>
+            ) : (
+              <>
+                {!searchResults?.length && (
+                  <EmptyResultsContainer
+                    css={css`
+                      margin-top: 200px;
+                    `}
+                  >
+                    {firstRenderComplete ? <EmptyResults type={'searchLandingPage'} /> : <EmptyPlaceholder />}
+                  </EmptyResultsContainer>
+                )}
               </>
             )}
-            {showMobileFilters && isTabletOrMobile && <MobileFilters />}
           </SearchResultsContainer>
         ) : (
           <>
+            {newSearchInput && (
+              <SearchResultsContainer>
+                <HeaderContainer>
+                  <H1 style={{ color: '#00684A', paddingBottom: '40px' }}> Search Results</H1>
+                  <SearchInput
+                    value={searchField}
+                    placeholder="Search"
+                    onSubmit={(event) => {
+                      setSearchResults([]);
```

Comment:
this is causing a infinite load when searching for same term, due to useEffect not firing

```suggestion
                      const newValue = event.target[0]?.value;
                      if (newValue === searchTerm) return;
                      setSearchResults([]);
```

can also setSearchTerm(newValue) here